reconcile/quay_robot_accounts.py

```
@@ -0,0 +1,400 @@
+
+def get_current_robot_accounts(
+    quay_api_store: QuayApiStore,
+) -> dict[tuple[str, str], list[RobotAccountDetails]]:
+    """Fetch current robot accounts from Quay API for all organizations"""
+    current_state = {}
+
+    for org_key, org_info in quay_api_store.items():
+        try:
+            robots = org_info["api"].list_robot_accounts()
+            current_state[org_key.instance, org_key.org_name] = robots or []
+        except Exception as e:
```

Ruff (0.15.21)

[warning] Do not catch blind exception: `Exception`

(BLE001)